src/ts/vault.ts

```diff
@@ -33,7 +30,7 @@ export async function grantRequiredRoles(
   for (const name in VAULT_INTERFACE.functions) {
     await authorizer.grantRole(
       ethers.utils.solidityKeccak256(
-        ["address", "bytes4"],
+        ["uint256", "bytes4"],
         [vaultAddress, VAULT_INTERFACE.getSighash(name)],
```

Comment:
Why is an address hashed as a number now? Is there a difference in the result by the way, considering that both are padded left?

Reply:
Its now `keccak256(uint256(address), sighash)` in the code. I think they just changed it to avoid extra bytecode generated because of masking (they were super tight on bytecode before).